src/stories/molecules/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,27 @@
+import { type ReactElement, type ReactNode } from 'react';
+
+import './breadcrumbItem.css';
+
+export interface BreadcrumbItemProps {
+  href?: string;
+  isCurrent?: boolean;
+  children: ReactNode;
+}
+
+export const BreadcrumbItem = ({
+  isCurrent = true,
+  href,
+  children,
+}: BreadcrumbItemProps): ReactElement => {
+  return (
+    <li className="breadcrumbItem">
+      {href && !isCurrent ? (
```

Comment:
[imo] 必須じゃないけどこっちの方が見やすそう！
```suggestion
      {(href && !isCurrent) ? (
```